Shared/Navbar.razor.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Microsoft.AspNetCore.Components;
+using UI.Services;
+
+namespace UI.Shared
+{
+    public partial class Navbar
+    {
+        private Guid _user = new Guid();
+        [Inject] private NavigationManager MyNavigationManager { get; set; }
+        [Inject] private UserService UserService { get; set; }
+
+        protected override void OnInitialized()
+        {
+            base.OnInitialized();
+            Console.WriteLine(MyNavigationManager.Uri);
```

Comment:
is this supposed to be here ?

Reply:
No but already removed in login-register branch